README.md

```diff
@@ -1,42 +1,34 @@
-# Nuance Dragon Medical SpeechKit
-
-> Develop and Troubleshoot [Dragon Medical SpeechKit](https://www.nuance.com/healthcare/developers.html) healthcare speech recognition for developers.
-
-## Overview
-
-### Built for Healthcare
-
-With decades of healthcare experience, an extensive industry footprint, and our highly scalable voice platform, Nuance offers SDKs to match the needs and priorities of healthcare clients.
-
-### Powered by Machine Learning
-
-Nuance advanced voice and language technologies are based on billions of end‑user interactions. Our hosted platform learns at scale, capturing more than 300 million minutes of voice dictation every year for a smarter, faster, more responsive experience.
-
-### Designed for Rapid Navigation
-
-By offering a unique blend of sophistication and simplicity, we can drastically enhance your healthcare app without complicated coding. Complete integration of speech dictation and voice-enabled workflows in minutes or hours, not days or weeks.
-
-### Enabled with Professional-grade Features
-
-Advanced command and control and text-to-speech capabilities create a more natural and responsive user experience. Beyond simply recording speech, our SDKs and services drive secure, intelligent voice-enabled workflows with features such as Auto-texts, custom medical vocabularies, voice correction capabilities and more.
-
-### Established Across the Globe
-
-Available in a range of hosted and on-premise configurations, our platform supports 20 languages for developers and clients in more than 30 countries.
-
-### Committed to End-to-End Security
-
-## Installation
-### Swift Package Manager
-- To add a package dependency to your Xcode project, select File > Swift Packages > Add Package Dependency 
-- Enter repository URL - https://github.com/nuance-communications/Dragon-Medical-SpeechKit-iOS
-- Confirm the version and let Xcode resolve the package
-- Add package to your Target
-
-## Getting Started
-
-Sign up for a 90-day [free trial license](https://www.nuance.com/healthcare/developers.html)
-
-## License
- 
- Refer [License](./LICENSE.md)
+# Nuance Dragon Medical SpeechKit
+Dragon Medical SpeechKit enables industry-leading healthcare speech recognition services for mobile and desktop application developers. For more information, documentation and other resources, register at: https://www.nuance.com/healthcare/developers.html
+
+This GitHub repository provides the Swift package for Dragon Medical SpeechKit (iOS edition).
+
+## Installation
+To add the Swift package to your project, do the following:
+     1.     In the Xcode **Projects and Targets** list, select your project, then select the **Swift Package** tab.
+     2.     Select + in the lower left corner of the **Swift Packages** section to add a new package.
+     3.     In the search bar, enter the repository URL: https://github.com/nuance-communications/Dragon-Medical-SpeechKit-iOS
```

Comment:
Another way to add Swift Package Dependency :
From menu, select File > Swift Packages > Add Package Dependency and enter its repository URL.